content/en/docs/concepts/security/service-accounts.md

```diff
@@ -0,0 +1,200 @@
+that Kubernetes grants to all authenticated accounts. If you delete the default
+ServiceAccount object in a namespace, the {{<glossary_tooltip text="control plane" term_id="control-plane">}}
+replaces it with a new one.
+
+If you deploy a Pod in a namespace, and you don't manually assign a
+ServiceAccount object to the Pod, Kubernetes assigns the `default` service
+account for that namespace to the Pod. You can change this behavior by setting
+the `automountServiceAccountToken` field in your Pod specification to `false`.
+
+## Use cases for Kubernetes service accounts
+
+As a general guideline, you can use service accounts to provide identities in
+the following scenarios:
+
+* Your Pods need to communicate with the Kubernetes API server, for example in
+  situations such as the following:
+  *  Providing read-only access to sensitive information stored in Secrets.
+  *  Granting [cross-namespace access](TODO), such as allowing a Pod in namespace
+     `example` to read, list, and watch for Lease objects in the
+     `kube-node-lease` namespace.
+* Your Pods need to communicate with an external service. For example,
+  [Authenticating to a private image registry using an `imagePullSecret`](/docs/tasks/configure-pod-container/configure-service-account/#add-imagepullsecrets-to-a-service-account).
+* An external service needs to communicate with the Kubernetes API server. For
+  example, authenticating to the cluster as part of a CI/CD pipeline.
+
+## How to use service accounts
+
+To use a Kubernetes service account, you do the following:
+
+1. Create the ServiceAccount object in the Kubernetes API using a Kubernetes
+   client like `kubectl` or a manifest that defines the object.
+1. Grant permissions to the ServiceAccount object using an authorization
+   mechanism such as RBAC.
+1. Assign the ServiceAccount object to Pods during Pod creation.
+
+For instructions, refer to
+[Configure Service Accounts for Pods](/docs/tasks/configure-pod-container/configure-service-account/).
+
+### Grant permissions to a ServiceAccount
+
+You can use the built-in Kubernetes
+[role-based access control (RBAC)](/docs/reference/access-authn-authz/rbac/)
+mechanism to grant the minimum permissions required by each service account.
+You create a *role*, which grants access, and then *bind* the role to your
+ServiceAccount. RBAC lets you define a minimum set of permissions so that the
+service account permissions follow the principle of least privilege. Pods that
+use that service account don't get more permissions than are required to
+function correctly.
+
+#### Cross-namespace access using a ServiceAccount
+
+You can use RBAC to allow service accounts in one namespace to perform actions
+on resources in a different namespace in the cluster. For example, consider a
+scenario where you have a service account and Pod in the `dev` namespace and
+you want your Pod to see Jobs running in the `batch` namespace. You could
```

Comment:
nit: to avoid confusion, maybe use a different example name than the API group for jobs?

```suggestion
you want your Pod to see Jobs running in the `maintenance` namespace. You could
```